### PR TITLE
Adding in content table pointers

### PR DESCRIPTION
**Pull request by jakub-pomykala**

### CODE REVIEW

File: posts/2021-08-17-KevinSutter_MeetTheTeam.adoc [hunk 1]
```diff
@@ -66,11 +67,12 @@ I find open-source development very enjoyable and very educational.  Everyday I
 
 Schedules are also a little more flexible in the open-source environment.  We still set goals and deadlines.  However, since this is an open-source effort, we are dependent on the time and resources available from any given organization.  We still find ourselves working long hours at times, but it's more because of our desire and commitment to completing the task rather than because an executive needs something for a customer presentation by Monday morning.
 
-
+[#experiance]
```
Comment (gljrobinson) [inline posts/2021-08-17-KevinSutter_MeetTheTeam.adoc]: experience rather than experiance

File: posts/2021-08-17-KevinSutter_MeetTheTeam.adoc [hunk 2]
```diff
@@ -43,19 +43,20 @@ Due to my involvement with these two key Working Groups at the Foundation, I was
...
-* Why is JakartaEE an important open source project? What does it offer? How did this project come about? How did it evolve from EE4J?
-What is it like to create a specification for an open-source project like this? How does it differ to working on code?
-* How did you become a project lead for JakartaEE? What is involved with being a project lead for an open source project like this?
-* As a fan of enterprise Java and it's related open source projects, why would you recommend this language and it's related projects for those seeking to build cloud-native enterprise applications?
-* What tips would you give for someone who is interested in getting involved in open source? Any specific tips for the JakartaEE or MicroProfile communities?
-* Now a fun question -- What activities do you enjoy outside of work?
 
+* <<encouraged, What encouraged you to get started with open source projects? How does working on an open-source community project like this compare to working on a proprietary project?>>
+* <<experiance, How has your experience working on open-source project affected or influenced your work at IBM?>>
```
Comment (gljrobinson) [inline posts/2021-08-17-KevinSutter_MeetTheTeam.adoc]: change this too to match anchor

**Review by gljrobinson — CHANGES_REQUESTED** [review]
Looks great, just a small spelling error is all. I've added two comments for this.

File: posts/2021-08-17-KevinSutter_MeetTheTeam.adoc (hunk 1 above)
Comment (jakub-pomykala) [reply posts/2021-08-17-KevinSutter_MeetTheTeam.adoc]: Done

File: posts/2021-08-17-KevinSutter_MeetTheTeam.adoc (hunk 2 above)
Comment (jakub-pomykala) [reply posts/2021-08-17-KevinSutter_MeetTheTeam.adoc]: Done

**Review by gljrobinson — APPROVED** [review]
LGTM

**Pull request merged**